Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);

```

Comment:
**Suggestion**: The Scanner instance 'sc' is created but never used, which is unnecessary overhead. Remove this line to clean up the code. Also, since the array 'arr' is identical in both methods, consider defining it as a class-level static variable to avoid duplication. <br> **Code Suggestion**: 
 ```
 -        Scanner sc = new Scanner(System.in);
 -        int[] arr = new int[]{1, 2, 3, 4, 5};
 +        // Removed unused Scanner instance and duplicate array definition.
 ```